web/src/features/charts/DataSources.tsx

```diff
@@ -53,6 +88,19 @@ export function DataSources({
   );
 }
 
+function EmissionFactorTooltip({ t }: { t: TFunction<'translation', undefined> }) {
+  return (
+    <Portal.Root className="pointer-events-none absolute left-0 top-0 z-50 flex h-full w-full flex-col content-center items-center justify-center bg-black/20 pb-40">
+      <div className="dark:border-1 relative mx-6 h-auto min-w-64 rounded-xl border bg-zinc-50 p-4 text-left text-sm opacity-80 shadow-md dark:border-gray-700 dark:bg-gray-900">
+        {t('country-panel.emissionFactorDataSourcesTooltip')}
+      </div>
+      <button className="p-auto pointer-events-auto mt-2 flex h-10 w-10 items-center justify-center self-center rounded-full border bg-zinc-50 text-black shadow-md">
+        <HiXMark size="24" />
+      </button>
```

Comment:
Tiny thing but this button should probably change colour depending on the mode that is used so it matches the other backgrounds.

Right now it looks like this in dark mode:
<img width="383" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/492e88d7-b7b8-4ec2-931c-f00024714d16">
And like this in light mode:
<img width="383" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/4b1700d3-fe23-4184-b482-057e2c3fbaea">

Reply:
It is a part of the design, so it is actually on purpose. Maybe @Alportan can elaborate on why:)

Reply:
@VIKTORVAV99 after some testing on dark theme, I discovered that people didn't really see the button because of the lack of contrast. So I would keep it white, no matter the selected theme. It might look like a bug, but it's a feature 😄

Reply:
Hmm...

I guess that's fine then but it looks weird 😅